web/quiz_views.py

```
@@ -851,3 +1083,127 @@ def quiz_analytics(request, quiz_id):
     # All context variables are set correctly
 
     return render(request, "web/quiz/quiz_analytics.html", context)
+
+
+@login_required
+def student_exam_correction(request, course_id, quiz_id, user_quiz_id):
```

Ruff (0.8.2)

`student_exam_correction` is too complex (19 > 10)

(C901)